### DISCUSSION TITLE
Provide a way to filter icons by library in the icon selection

### DISCUSSION THREAD

**Discussion by JohannesPertl** [directus/directus#20619] [Feature Requests]
### Summary

Our svelte frontend uses material icons. Currently the CMS admin has no way of knowing if the selected icon is a material icon and will display on the frontend. 

![image](https://github.com/directus/directus/assets/45853628/5f1bab24-c6ec-4dfb-abdf-60e30bd473a4)

A simple checkbox to only show material icons would solve this annoying problem.

### Motivation

Usability improvement and therefore better adoption of directus as a general cms.

### Detailed Design

A filter by icon library in the directus backend and a checkbox in the icon selection frontend.

### Must Have:
- Filter by icon library in the icon selection field.

### Could Have:
- A setting to only enable icons of a specific library.

### Drawbacks

None.

### Alternatives

A directus npm package that directly provides all icons regardless which library is used by directus.

### Adoption Strategy

No breaking change, simple but useful usability enhancement

**Comment by rijkvanzanten** [comment]
Directus uses the updated Material _Symbols_, not Material _Icons_. If you update the icon font on your front-end project all icons should be available 🙂
